submissions/NJul/dom-api-task/app.js

```diff
@@ -0,0 +1,98 @@
+const advices = [
+  {
+    id: 1,
+    title: `Designate a study area`,
+    content: `The best study spot is one that is quiet, well-lit, and in a low-traffic area. Make sure there is a clear workspace to study and write on. Everyone’s needs are different, so it is important you find a spot that works for you.`,
+    source: ``,
+  },
+  {
+    id: 2,
+    title: `Talk to teachers`,
+    content: `Teachers are there to help you do your best. Talk to your teacher and ask for clarification or extra help if you need it before your test. Taking the initiative to ask for help goes a long way with teachers!`,
+    source: ``,
+  },
+  {
+    id: 3,
+    title: `Study with a group`,
+    content: `Working with classmates encourages an interactive environment to keep you engaged. This gives you a chance to test your knowledge with others, quiz each other on the content, and help boost each other’s confidence.`,
+    source: ``,
+  },
+  {
+    id: 4,
+    title: `Ask questions if you don't understands`,
+    content: `Raise your hand and ask questions if you don’t understand something. If you don’t feel comfortable asking in front of everyone, write yourself a reminder to talk to the teacher after class.`,
+    source: `<a href="https://www.oxfordlearning.com/how-to-study-effectively/" target="_blank">Source</a>`,
+  },
+];
+
+document.addEventListener('DOMContentLoaded', () => {
+  const hamburgerBtn = document.getElementById('hamburgerBtn');
+  const navBar = document.getElementById('navBar');
+
+  hamburgerBtn.addEventListener('click', () => {
+    navBar.classList.toggle('open');
```

Comment:
This leads to a situation where `class` attribute may have no value at certain points. This is not not allowed by html specification, since only boolean attributes (like "checked") may have no value. Can you think of a different approach to class toggling?